packages/fetchai/connections/ledger/ledger_dispatcher.py

```diff
@@ -135,6 +136,35 @@ def get_balance(
             )
         return response
 
+    def get_state(
+        self, api: LedgerApi, message: LedgerApiMessage, dialogue: LedgerApiDialogue,
+    ) -> LedgerApiMessage:
+        """
+        Send the request 'get_state'.
+
+        :param api: the API object.
+        :param message: the Ledger API message
+        :return: None
+        """
+        callable = getattr(api, message.callable)
+        args = tuple(message.kwargs.body.values())
+        data = callable(*args)
```

Comment:
If the ledger api has generic `get_state` than here we can be explicit and move the generics down the the ledger api (see above)